src/components/Profile/Profile.css

```diff
@@ -0,0 +1,104 @@
+.profile {
+  display: grid;
+  grid-template-columns: minmax(auto, 250px) auto;
+  margin-top: 20px;
+}
+
+.profile__search {
+  margin-left: 20px;
+}
+
+.profile__search_input {
+  color: #fff;
+  margin-left: 10px;
+}
+
+.profile__search_album {
+  margin-top: 20px;
+  margin-left: 10px;
+  background-color: aliceblue;
+  border: none;
+  border-radius: 20px;
+}
+
+.profile__search_button {
+  background-color: aliceblue;
+  border: none;
+  margin-left: 10px;
+  border-radius: 20px;
+}
+
+.profile__sidebar {
+  background-color: black;
+  border-radius: 30px;
+  padding-bottom: 300px;
+}
+
+.profile__album {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.profile__album_card {
+  width: 150px;
+  list-style-type: none;
+  margin-left: 20px;
+  margin-top: 20px;
+}
+.album__bookmark {
+  background-color: #fff;
+  border-radius: 15px;
+  position: absolute;
+  margin-left: 120px;
+  margin-top: 4px;
+}
+
+.album__description {
+  display: flex;
+}
+
+.profile__album_name {
+  font-family: roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  top: 15px;
+  padding: 2px 12px;
+  color: #fff;
+  font-size: 15px;
+  margin-bottom: 2px;
+}
+
+.profile__album_artist {
+  font-family: roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  top: 15px;
+  padding: 2px 12px;
+  color: #fff;
+  font-size: 15px;
+  margin-top: 2px;
+}
+.profile__album_image {
+  width: 100%;
+  border-radius: 19px;
+  border: 1px solid;
+  color: rgba(0, 0, 0, 0.3);
+}
+
+.Profile__album_placeholder {
```

Comment:
`Profile`


All `BEM` class names should be in **lower case**, to be consistent